clu/src/lib.rs

```diff
@@ -84,11 +82,11 @@ fn linux_setup_exit_tunnel(config: Arc<RwLock<settings::RitaSettingsStruct>>) ->
         details.own_internal_ip,
         details.netmask,
     )?;
-    KI.set_route_to_tunnel(&details.server_internal_ip);
+    KI.set_route_to_tunnel(&details.server_internal_ip).expect("Failed to set default route");
```

Comment:
Why not `?`?

Reply:
ok this is a good case to propogate up, you could conceivably clean up rotues properly and handle it.